### PR TITLE
Fixes clickdrag on shinai bag

### LINKED ISSUE #3644: Shinai bag cannot be taken off normally.
[BUG - MINOR]

**Describe the bug**
The shinai bag cannot be taken off by clickdragging it to your hand like other bags, only by swapping it for another backpack or having someone else remove it.

**To Reproduce**
1. put on a shinai bag
2. try to clickdrag it into your hand

**Expected behavior**
the bag should be moved from your back slot to your hand.

### PR DESCRIPTION
**Pull request by MetricDuck**
[BUG - MINOR]
## About the PR 
Shinai bags cannot be taken off by clickdragging to hand because they do not inherit the relevant proc from /obj/item/storage - they are of path /obj/item/shinai_bag. They can still be removed by clickdragging to floor.

This PR adds the relevant override to MouseDrop to enable clickdragging to hand.

## Why's this needed? 
Fixes #3644

### CODE REVIEW

File: code/obj/item/kendo.dm [hunk 1]
```diff
@@ -240,6 +240,23 @@
...
+				if (S.id == "rhand")
+					if (!usr.r_hand)
+						usr.u_equip(src)
+						usr.put_in_hand_or_drop(src, 0)
+				else
+					if (S.id == "lhand")
+						if (!usr.l_hand)
+							usr.u_equip(src)
+							usr.put_in_hand_or_drop(src, 1)
+				return
```
Comment (ZeWaka) [inline code/obj/item/kendo.dm:258]:
```suggestion
```
useless

Comment (MetricDuck) [reply code/obj/item/kendo.dm]: return removed

Comment (ZeWaka) [reply code/obj/item/kendo.dm]: you can resolve conversations with the button press btw

Comment (MetricDuck) [reply code/obj/item/kendo.dm]: Ah I didn't realise the reviewer got a ping! Had it in my mind it was for the reviewer to resolve as a "I agree these changes resolve my comments"

**Review by ZeWaka — COMMENTED** [review]
why are they not /obj/item/storage?

**Comment by MetricDuck** [comment]
> why are they not /obj/item/storage?

I would guess because the code was originally based off of katana sheath's, which are obj/item/katana_sheath. The bag could be converted to obj/item/storage/shinai_bag with two slots filtered only to accept shinai, though other than the HUD I can't think of any additional behaviour that wouldn't need to be immediately overridden.

File: code/obj/item/kendo.dm [hunk 2]
```diff
@@ -240,6 +240,22 @@
 		else
 			..()
 
+	MouseDrop(atom/over_object, src_location, over_location)
```
Comment (pali6) [inline code/obj/item/kendo.dm:243]:
```suggestion
	mouse_drop(atom/over_object, src_location, over_location)
```

File: code/obj/item/kendo.dm [hunk 3]
```diff
@@ -240,6 +240,22 @@
...
+		..()
+		var/atom/movable/screen/hud/S = over_object
+		if (istype(S))
+			playsound(src.loc, "rustle", 50, 1, -5)
+			if (!usr.restrained() && !usr.stat && src.loc == usr)
```
Comment (pali6) [inline code/obj/item/kendo.dm:248]:
```suggestion
			if (can_act(usr) && src.loc == usr)
```

**Pull request merged**